First/src/tutorials/StackDemo.java

```diff
@@ -0,0 +1,26 @@
+package tutorials;
+
+import java.util.Stack;
+
+public class StackDemo {
+
+	public static void main(String[] args)
+	{
+		Stack<String> stack = new Stack<>();
```

Comment:
Can name it as Countries as well

Reply:
Named it as StackOperations because I already have another program named as Countries